client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -259,6 +261,14 @@ export class DomainDetailsForm extends PureComponent {
 		return cartItems.getDomainRegistrationsWithoutPrivacy( this.props.cart ).length === 0;
 	}
 
+	canDisplayAddressFieldset() {
```

Comment:
So we tend to deconstruct up front, like
`const { countryCode, ... } = this.state;` (I think because we do it that way with props)